File: server/dotnet/sdk-dotnet-example/Infrastructure/Mappers/HostedTokenizationMapper.cs
using Business.DTOs.HostedTokenizations;
using OnlinePayments.Sdk.Domain;

namespace Infrastructure.Mappers;

public static class HostedTokenizationMapper
{
    public static GetHostedTokenizationResponseDto Map(CreateHostedTokenizationResponse? response)
    {
        return new()
        {
            HostedTokenizationId = response?.HostedTokenizationId,
            HostedTokenizationUrl = response?.HostedTokenizationUrl
        };
    }
}

File: server/java/src/main/java/com/onlinepayments/example/sdk/mappers/HostedTokenizationMapper.java
package com.onlinepayments.example.sdk.mappers;

import com.onlinepayments.domain.CreateHostedTokenizationResponse;
import com.onlinepayments.example.service.dtos.hostedtokenization.GetHostedTokenizationResponseDto;
import org.mapstruct.Mapper;
import org.mapstruct.Mapping;
import org.mapstruct.Mappings;

@Mapper(componentModel = "spring")

public interface HostedTokenizationMapper {
    @Mappings({
            @Mapping(target = "hostedTokenizationId", source = "hostedTokenizationId"),
            @Mapping(target = "hostedTokenizationUrl", source = "hostedTokenizationUrl"),
    })
    GetHostedTokenizationResponseDto toResponseDto(CreateHostedTokenizationResponse response);
}
